staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -145,9 +145,16 @@ type GenericAPIServer struct {
 	preShutdownHooksCalled bool
 
 	// healthz checks
-	healthzLock    sync.Mutex
-	healthzChecks  []healthz.HealthzChecker
-	healthzCreated bool
+	healthzLock                sync.Mutex
+	healthzChecks              []healthz.HealthzChecker
+	healthzChecksLocked        bool
```

Comment:
"Locked" makes it sound like it's got to do with the mutex, I suggest "Installed" or "Finalized"?